File: FastScript/Demos/C++Demo/C++Script/array.cpp
/***************************/
/* FastScript v1.0         */
/* Arrays demo             */
/***************************/

int i, j;
string ar[10], s;


void OK(string s)
{
  ShowMessage(s);
}

{
  for(i = 0; i < 10; i++)
    ar[i] = IntToStr(i);

  s = "";
  for(i = 0; i< 10; i++)
    s += ar[i];

  OK(s);
}
    

File: FastScript/Demos/Main/Samples/BasicScript/array.vb
'**************************
' FastScript v1.0          
' Arrays demo              
'**************************

dim i, j
dim ar[10]
dim s

for i = 0 to 9
  ar[i] = IntToStr(i)
next

s = ""
for i = 0 to 9 
  s = s + ar[i]
next

ShowMessage(s)


    